style.css

```diff
@@ -0,0 +1,238 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  box-shadow: 'poppins', sans-serif;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  width: 100%;
+}
+
+header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 50px;
+}
+
+.head-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  height: 100%;
+}
+
+.head-content .logo {
+  font-size: 20px;
+  font-weight: normal;
+}
+
+.head-content .nav {
+  padding-top: 12px;
+  width: 50px;
+  height: 100%;
+  position: relative;
+}
+
+.head-content .nav i {
+  font-size: 20px;
+  margin: 5px 0 5px 0;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  gap: 50px;
+}
+
+section .presentation {
+  background-image: url(assets/capt.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-origin: content-box;
+  background-size: cover;
+  margin-top: 60px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+section .presentation h1 {
+  text-align: center;
+}
+
+section .presentation h2 {
+  color: rgb(6, 141, 6);
+  font-size: 1.4em;
+  text-align: center;
+  font-weight: bold;
+}
+
+section .presentation .text {
+  text-align: center;
+  margin: 50px 0 50px 0;
+}
+
+section .presentation .icons {
+  align-self: center;
+  row-gap: 10px;
+  list-style-type: none;
+  display: flex;
+}
+
+section .presentation .icons i {
+  margin: 10px;
+  color: rgb(51, 49, 49);
+}
+
+a {
+  text-decoration: none;
+  color: rgb(51, 49, 49);
+}
+
+section .works {
+  display: grid;
+  height: auto;
+  width: 100%;
+  gap: 50px;
+}
+
+section .works .more {
+  justify-self: center;
+  border: solid 1px rgb(168, 167, 167);
+  padding: 10px;
+  border-radius: 50px;
+  width: 52px;
+  font-size: 25px;
+  color: #172b4d;
+  text-align: center;
+}
+
+section .works .works-title {
+  color: #172b4d;
+  font-size: 40px;
+  font-weight: 700;
+  text-align: center;
+  justify-self: center;
+}
+
+section .works .columns {
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+  gap: 50px;
+  width: 100%;
+}
+
+section .works .columns .card {
+  display: grid;
+  background-color: #e5e5e5;
+  border: solid 1px rgb(168, 167, 167);
+  border-radius: 8px;
+  width: 100%;
+  height: auto;
+}
+
+section .works .columns .card .works-heading {
+  height: 250px;
+}
+
+section .works .columns .card .works-footer {
+  background: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 8px;
+  gap: 15px;
+  height: auto;
+  border-radius: 8px;
+}
+
```

Comment:
- [ ] Kindly fix the font, weight, and color to make it like a Figma design
- [ ] Kinldy make changes in all the part
### your design
![your design](https://user-images.githubusercontent.com/81727988/176916375-ce84a474-00a1-408e-9b5d-de3b1ff61a27.png)
### Figma design
![his](https://user-images.githubusercontent.com/81727988/176916418-dd16a898-8742-4929-bb0b-eda3f1c04af7.png)